ui/pages/swaps/awaiting-signatures/awaiting-signatures.js

```diff
@@ -94,57 +93,40 @@ export default function AwaitingSignatures() {
         <Box marginTop={3} marginBottom={4}>
           <PulseLoader />
         </Box>
-        <Typography
+        <Text
           color={TextColor.textDefault}
-          variant={TypographyVariant.H3}
+          variant={TextVariant.headingMd}
+          as="h3"
         >
           {headerText}
-        </Typography>
+        </Text>
         {needsTwoConfirmations && (
           <>
-            <Typography
-              variant={TypographyVariant.paragraph}
-              boxProps={{ marginTop: 2 }}
-              fontWeight={FONT_WEIGHT.BOLD}
-            >
-              {t('swapToConfirmWithHwWallet')}
-            </Typography>
+            <Text variant={TextVariant.bodyMd} boxProps={{ marginTop: 2 }}>
+              <strong>{t('swapToConfirmWithHwWallet')}</strong>
+            </Text>
```

Comment:
```suggestion
            <Text variant={TextVariant.bodyMdBold} boxProps={{ marginTop: 2 }}>
              {t('swapToConfirmWithHwWallet')}
            </Text>
```
Please refer to https://github.com/MetaMask/metamask-extension/pull/18752#discussion_r1174597778